smartsim/_core/control/controller.py

```diff
@@ -298,35 +310,43 @@ def _launch(self, manifest: Manifest) -> None:
         :param manifest: Manifest of deployables to launch
         :type manifest: Manifest
         """
-        orchestrator = manifest.db
-        if orchestrator:
+
+        orchestrators = manifest.dbs
```

Comment:
It looks like `orchestrators` is only used on line 316. Consider removing variable and simply iterate over `manifest.dbs`